src/test/scala/com/fulcrumgenomics/umi/DuplexConsensusCallerTest.scala

```diff
@@ -289,13 +289,8 @@ class DuplexConsensusCallerTest extends UnitSpec {
     // Add the original UMI bases to each read
     builder.foreach { rec =>
       val mi = rec[String](MI)
-      // first of pair ABs and second of pair BAs
-      if ((rec.firstOfPair && mi.endsWith("/A")) || (rec.secondOfPair && mi.endsWith("/B"))) {
-        rec(RX) = "AAT-CCG"
-      }
-      else {
-        rec(RX) = "CCG-AAT"
-      }
+      // all "/A" have the same UMI bases, while the "/B" bases have them swapped
+      if (mi.endsWith("/A")) rec(RX) = "AAT-CCG" else rec(RX) = "CCG-AAT"
```

Comment:
Does this test still pass?  I see below that we check the RX tag on the output reads and expect them to differ.  Wouldn't we expect them to be the same after this change?